index.js

```diff
@@ -427,6 +428,15 @@ class _ChartVisualization extends Visualization {
         slotContainer.style.width = "100%"
 
         this.chart = await this.getChart()
+
+        // callback for CSV download
+        top.document.getElementById(`chart_${this.slot}_download`).addEventListener("click", () => {
+            console.log("Export to CSV triggered.")
+            this.downloadCSV({
+                filename: "chart-data.csv",
+            })
+        });
```

Comment:
I would have liked to put this in the callbacks section of `ui.js`, but I couldn't find a way to get a handle for what is `this` here.